lib/compress/zstdmt_compress.c

```diff
@@ -452,6 +456,8 @@ struct ZSTDMT_CCtx_s {
     ZSTD_CCtx_params params;
     size_t targetSectionSize;
     size_t targetPrefixSize;
+    buffer_t inBuffSpace;
+    size_t inBuffPos;
```

Comment:
`inBuffPos` seems to indicate how much data has been filled so far within `inBuffSpace`.
But since we also have some `inBuff` slices present the code, this feels confusing, as the naming implies the 2 are related.
Recommendation : change the name to `inBuffSpacePos`, or better make this value a member of `inBuffSpace` (which will have to change type).

Note : this is similar to what you did with `range_t`, and helps clarity.